deployments/polygon.json

```diff
@@ -50,5 +50,6 @@
   "GenericSwapFacetV3": "0x31a9b1835864706Af10103b31Ea2b79bdb995F5F",
   "StargateFacetV2": "0x6e378C84e657C57b2a8d183CFf30ee5CC8989b61",
   "ReceiverStargateV2": "0x1493e7B8d4DfADe0a178dAD9335470337A3a219A",
-  "LiFiDEXAggregator": "0x6140b987d6B51Fd75b66C3B07733Beb5167c42fc"
-}
+  "LiFiDEXAggregator": "0x6140b987d6B51Fd75b66C3B07733Beb5167c42fc",
+  "Permit2Proxy": "0x5c6e208468213999CB0Be4bAB6d1492c3139A4EE"
+}
```

Comment:
That is not correct.
Also the LiFiDEXAggregator is not part of this PR. This address in the main branch already. Therefore you should not be analyzing this at all.
I've asked you both of these things in the past to note as learnings:

1) only check the parts that are touched by this PR
2) make sure that your way/script to check if a contract has bytecode deployed actually works since you have been creating a lot of false negatives for this issue. May I suggest that you use 'case code 'ADDRESS' 'RPC-URL'' from the foundry framework to check if a contract has code deployed?